### PR TITLE
Rld 22 select inputs

### PR DESCRIPTION
**Pull request by ainouzgali**

### CODE REVIEW

File: apps/web/src/design-system/select/Select.tsx [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+import CheckboxItem from './CheckboxItem';
+import useStyles from './Select.styles';
+import { inputStyles } from '../config/inputs.styles';
+
+interface ISelectProps {
+  data: (string | { value: string; label?: string })[];
+  label?: React.ReactNode;
+  placeholder?: string;
+  description?: string;
+  type?: 'Checkbox' | 'Label';
```
Comment (scopsy) [inline apps/web/src/design-system/select/Select.tsx:21]: @ainouzgali can we make the types lowercased? 
```suggestion
  type?: 'checkbox' | 'label';
```

Comment (ainouzgali) [reply apps/web/src/design-system/select/Select.tsx:21]: sure :) I think I'll also change it to multiselect | select(default) , wdyt?

File: apps/web/src/design-system/tempCheckbox/Checkbox.tsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import React from 'react';
```
[the remark this answers is not in the document]
Comment (ainouzgali) [reply apps/web/src/design-system/tempCheckbox/Checkbox.tsx:1]: no no, I knew a pr for checkbox is coming. Just wanted to play with the checkbox dropdown. 
Wanted to consult on that dropdown, It's just for the draft.

**Review by scopsy — APPROVED** [review]
10/10 🤩

**Pull request merged**